package.json

```diff
@@ -4,6 +4,8 @@
   "description": "Memory game",
   "main": "main.js",
   "scripts": {
+    "start": "parcel index.html",
```

Comment:
nowadays you don't have to use bundler to have classes in the separate files but `parcel` is a good choice